ASTProcessor/src/main/java/edu/usb/argos/ASTProcessor/visitor/core/interfaces/visitor/IAttributeAnalyzer.java

```diff
@@ -0,0 +1,14 @@
+package edu.usb.argos.ASTProcessor.visitor.core.interfaces.visitor;
+
+import edu.usb.argos.ASTProcessor.visitor.core.entities.method.AttributeInfo;
+import java.util.List;
+
+public interface IAttributeAnalyzer<T> {
+
+    List<AttributeInfo> visitAttribute(T ctx);
```

Comment:
Also, it would be better if you don't abbreviate again the parameter name.

Reply:
Done